test/integration/MenuList.spec.js

```diff
@@ -4,8 +4,8 @@ import React from 'react';
 import keycode from 'keycode';
 import { assert } from 'chai';
 import { spy } from 'sinon';
-import MenuList from 'src/Menu/MenuList';
-import MenuItem from 'src/Menu/MenuItem';
+import MenuList from 'src/MenuList/MenuList';
+import MenuItem from 'src/MenuItem/MenuItem';
```

Comment:
Wait, what?

Reply:
Bah, sorry, stray change from the `flat-src-dir` experiment. I need to be more careful with what `git add .` adds (or doesn't - I see how that got missed from the other commit...)